### DISCUSSION TITLE
self-hosted api menu not working (response 401)

### DISCUSSION THREAD

**Discussion by reichert** [supabase/supabase#5603] [Questions]
Hi guys, I've installed supabase with docker-compose and I have the following problem:

1) when acessing https://example.com/project/default/api the inspector show me a 401 error when calling https://example.com/rest/v1/?apikey=eyJhbGciO...

2) If I put the same url directly in the browser a json with my models appears correctly.

I know this route is behind kong. I`m using a reverse proxy called traefik in front of kong as I do with every docker container installed. Any though about what is happening?

**Comment by dudlf** [comment]
Use jwt secret longer than 32 character. Postgrest requires password longer than 32 character.

see: 
https://postgrest.org/en/stable/tutorials/tut1.html?highlight=32#step-2-make-a-secret
https://github.com/PostgREST/postgrest/issues/977#issuecomment-329920851

**Reply by ryanhaticus** [reply]
Thanks. This solved all my problems!

**Comment by jameswong3388** [comment]
@reichert, did you manage to get it work with Traefik ?
